public/app/services/settingsService.js

```diff
@@ -13,17 +13,22 @@ angular.module('kibibitCodeEditor')
     var isFullscreen = false;
     var currentUndoManager = undefined;
     var currentEditor = undefined;
+    var modelist = ace.require('ace/ext/modelist');
+    var themelist = ace.require('ace/ext/themelist');
+    var editorSettings = new EditorSettings();
 
     /* EXPOSE SIMPLE VARS */
     // This are probably better off in state instead of settings. but they're here for now :-)
     settings.cursor = cursor;
     settings.currentUndoManager = currentUndoManager;
     settings.currentEditor = currentEditor;
+    settings.modelist = modelist;
+    settings.themelist = themelist;
 
     settings.__defineSetter__('isFullscreen', function(newValue) {
 
       console.assert(isBoolean(newValue), {
-        'message': 'isFullscreen should only be a boolean, but was given some other type',
+        'message': 'isFullscreen should be boolean',
```

Comment:
shouldn't this be "isFulscreen should be **a** boolean"?

I personally think the previous message give the right info. This message sounds to me as if someone could interpert this as if "In this particular situation, isFullscreen should be a boolean"

What do you think?